### DISCUSSION TITLE
Lesson 11 not able to solve this error "Unable to verify"

### DISCUSSION THREAD

**Discussion by avikat** [smartcontractkit/full-blockchain-solidity-course-py#1099] [Q&A] [answered]
Please help me with this. i think i am missing something

Waiting for https://api-rinkeby.etherscan.io/api to process contract...
Verification submitted successfully. Waiting for result...
Verification pending...
Verification pending...
Verification complete. Result: Fail - Unable to verify
Transaction sent: 0xe043f2ec96af8af9c569af64bf9dc9316511a5bdae72516d88e4ff31b96567a5
  Gas price: 1.200001874 gwei   Gas limit: 56992   Nonce: 66
  LinkToken.transfer confirmed   Block: 10194056   Gas used: 51811 (90.91%)

  LinkToken.transfer confirmed   Block: 10194056   Gas used: 51811 (90.91%)

Fund Transfer

**Comment by avikat** [comment]
Close this discussion please i think it is only two time thing.

**Comment by cromewar** [comment, marked as answer]
Closing this thread now as requested by the author.
